ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java

```diff
@@ -474,6 +478,8 @@ public String getType() {
   public void loadRtbRewardedAd(@NonNull MediationRewardedAdConfiguration mediationRewardedAdConfiguration,
       @NonNull MediationAdLoadCallback<MediationRewardedAd, MediationRewardedAdCallback> mediationAdLoadCallback) {
     Log.d(TAG, "loadRtbRewardedAd()...");
+    VungleInitializer.getInstance()
```

Comment:
We can remove this line due to we have done this in loadRewardedAd() method.

Reply:
@HuFangshuai thanks will verify